src/js/components/molecules/list-section/list-section.ts

```diff
@@ -87,7 +87,11 @@ export default class ListSection extends Vue implements IListSection {
 
     triggerEventClick (): void {
         if (this.expand && this.expand.clickEvent) {
-            this.$emitPublic(this.clickEventType, this.clickEventData);
+            try {
```

Comment:
this fix works, however i think we should make $emitPublic part of a constructor to force its presence.